test/integration/quota_project/quota_project_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package quota_project
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/url"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/utils"
+	"github.com/stretchr/testify/assert"
+	"golang.org/x/oauth2/google"
+)
+
+func TestQuotaProject(t *testing.T) {
+	quotaProjectT := tft.NewTFBlueprintTest(t)
+
+	quotaProjectT.DefineVerify(func(assert *assert.Assertions) {
+		quotaProjectT.DefaultVerify(assert)
+
+		projectID := quotaProjectT.GetStringOutput("project_id")
+
+		apis := gcloud.Run(t, fmt.Sprintf("services list --project %s", projectID))
```

Comment:
nit: newer versions of the framework has a Runf method which is a bit cleaner
```suggestion
		apis := gcloud.Runf(t, "services list --project %s", projectID)
```